Makefile

```diff
@@ -27,8 +27,8 @@ all: install
 
 build-linux:
 	@echo Build Linux amd64
-	env GOOS=linux GOARCH=amd64 $(GO) build -o $(AGENT) $(AGENT_ARGS)
-	env GOOS=linux GOARCH=amd64 $(GO) build -o $(API_SERVER) $(API_SERVER_ARGS)
+	env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -o $(AGENT) $(AGENT_ARGS)
```

Comment:
Isn't there any more magic like `-ldflags '-extldflags "-fno-PIC -static"' -buildmode pie -tags 'osusergo netgo static_build`? Just kidding, we should probably review this after static build is resolved by Go [here](https://github.com/golang/go/issues/26492).

Reply:
Yeah, mainly focusing on fixing the issue at hand more than providing a static build.

Reply:
Yeah, this doesn't _really_ do a static build. Just useful for cross-compiling, and disables any code that uses the tag `cgo`.